app/controllers/orders_controller.rb

```diff
@@ -2,4 +2,12 @@ class OrdersController < ApplicationController
   def index
     @orders = current_user.orders
   end
+
+  def show
+    @order = Order.find(params[:id])
+    if !current_user.orders.include?(@order)
```

Comment:
@Dpalazzari This is to make sure we only show orders belong to the logged in user (which is `current_user`). With our current implementation without this check, one user could log in, and enter the path of `/orders/:id` where the `:id` would refer to an order associated with another user. We are checking here that the order id we received through `params[:id]` is part of the collection of orders of the user that is logged in `current_user.orders`. FYI @njgheorghita @drod1000  Please let me know in case further explanation is necessary.